quad_control/scripts/utilities/jsonable.py

```diff
@@ -45,10 +45,15 @@ def to_string(cls, inner=dict()):
             arg = args[i+1]
             if arg in cls.inner.keys():
                 val = (inner[arg].__name__, inner[arg].to_string())
+            elif type(defs[i]) is np.ndarray:
+                val = str(list(defs[i]))
             else:
                 val = defs[i]
             arg_dic[arg] = val
-        return json.dumps(arg_dic, indent=4)
+        string = json.dumps(arg_dic, indent=4, separators=(', ', ':\n\t'))
```

Comment:
The separator between dictionary items is set as `:\n\t`, which is colon, new line and tab.